docs/dqx/docs/dev/contributing.mdx

```diff
@@ -93,17 +93,29 @@ make lint
 make test
 ```
 
-Setup required environment variables for executing integration tests and code coverage using the command line.
-Note that integration tests are run automatically when you create a Pull Request in Github.
-You can also run them from a local machine by configuring authentication to a Databricks workspace as below:
+### Local setup for integration tests and code coverage
+
+Note that integration tests and code coverage are run automatically when you create a Pull Request in Github.
+You can also trigger the tests from a local machine by configuring authentication to a Databricks workspace.
+You can use any Unity Catalog enabled Databricks workspace.
+
+#### Using terminal
+
+If you want to run integration tests and code coverage from your local machine, you need to set up the following environment variables:
 ```shell
 export DATABRICKS_HOST=https://<workspace-url>
 export DATABRICKS_CLUSTER_ID=<cluster-id>
-# set either service principal credentials
+
+# either use PAT token for authentication
+export DATABRICKS_TOKEN=<pat-token>
```

Comment:
Please don't recommend PATs - they're considered bad practice since the introduction of OAuth

Reply:
Thanks for the review, i will correct in the next PR